<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 208, in main
    return_code, report = execute_linter(
                          ~~~~~~~~~~~~~~^
        recipe=args.recipe,
        ^^^^^^^^^^^^^^^^^^^
    ...<6 lines>...
        verbose_flag=args.verbose,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 180, in execute_linter
    result = linter.lint(recipes, subdir, variant_config_files, exclusive_config_files, fix_flag)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 725, in lint
    msgs = self.lint_file(
        recipe_name,
    ...<3 lines>...
        fix=fix,
    )
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 831, in lint_file
    self.lint_recipe(
    ~~~~~~~~~~~~~~~~^
        recipe,
        ^^^^^^^
        fix=fix,
        ^^^^^^^^
    )
    ^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 891, in lint_recipe
    res = self.check_instances[check].run(recipe, fix)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 294, in run
    self.check_recipe(recipe)
    ~~~~~~~~~~~~~~~~~^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/check_syntax.py", line 36, in check_recipe
    has_constraints = constraints.search(spec)
TypeError: expected string or bytes-like object, got 'NoneType'

ERROR conda.cli.main_run:execute(127): `conda run conda-lint /tmp/abs_115735_kfm/clone` failed. (See above for error)
</details>

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -24,7 +24,7 @@ requirements:
     - setuptools
     - cython >=0.29
     # https://github.com/geopandas/pyogrio/blob/v0.10.0/setup.py#L24
-    - libgdal-core {{ libgdal }}
+    - libgdal-core {{ libgdal_core }}
     # Use versioneer 0.29 for py313 instead of 0.28
     # https://github.com/geopandas/pyogrio/blob/v0.10.0/pyproject.toml#L5
     - versioneer

